src/commands/cluster-slot-stats.json

```
@@ -0,0 +1,92 @@
+{
+    "SLOT-STATS": {
+        "summary": "Return array of slot usage statistics, of slots assigned to the current node",
+        "complexity": "O(N) where N is the total number of slots based on arguments. O(N log N) with ORDERBY subcommand",
+        "group": "cluster",
+        "since": "7.2.0",
+        "arity": -2,
+        "container": "CLUSTER",
+        "function": "clusterCommand",
```

Maybe let's go ahead and create a new function here. 
```suggestion
        "function": "clusterSlotStats",
```

Then we don't need to keep overloading the top level cluster container. We do this in a couple of places such as config-get.json

---

Understood, `clusterSlotStatsCommand` to be specific, including the `Command` suffix.
The new function will be defined under `cluster_slot.c/h`